tests/X86/FMA/VFMADDSD.S

```diff
@@ -16,19 +16,13 @@
 
 #if HAS_FEATURE_AVX
 
-// We have not yet been successful in modeling the so-called "infinite
-// precision" floating point that the FMA instructions supposedly use to
-// calculate the intermediary results of the fused-multiply operations. This 
-// means that tests with operand values that cannot be precisely stored using
-// binary floating point (e.g., 0.1, as opposed to 0.5 which can be 
-// stored precisely) will fail to match the output of the native FMA
-// instruction -- by exactly 1 bit.
 #define VFMADDSD_INPUTS_64                                             \
     0.0, 0.0, 0.0,    /* will evaluate to 0.0 */                       \
     1.0, 1.0, 1.0,    /* will evaluate to 2.0 */                       \
     2.0, 3.0, 4.0,    /* will evaluate to 10.0, 11.0, or 14.0 */       \
     2.5, 2.5, 2.5,    /* will evaluate to exactly 8.75 */              \
     1.75, 1.5, -2.25, /* test with negative values */                  \
```

Comment:
You'll want to test this type of thing in IDA, but these are equivalent to the following in assembly:
```asm
.quad 1.2
```
It's not clear to me that `1.2` will expand into the qword representation that actually represents the `1.2` value of a `double`.

Reply:
In practice they do (I've tested it at runtime) but if there's a risk they they won't _always_ compile this way, should they be specified as the hex?

Reply:
TIL :-) I don't think there's a risk because we control the compiler via `cxx-common`.